src/cgls/cgls.c

```diff
@@ -267,13 +258,12 @@ static int run(int argc, char *argv[]) {
                                                 return log_error_errno(r, "Failed to list cgroup tree: %m");
                                 }
 
-                                q = cg_split_spec(*name, &c, &p);
+                                q = cg_split_spec(*name, /* ret_controller = */ NULL, &p);
```

Comment:
i'd really prefer if we could start warning here if a controller is specified. maybe add a flags param CG_SPLIT_WARN_LEGACY or so?

Reply:
Yeah, the whole calling convention with a controller name doesn't make sense anymore. So I think we should warn if `c` is returned and ignore it.